tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = es, addons, devhub, reviewers, main, ui-tests, flake8, docs, assets
+envlist = es, addons, amo-locales-and-signin, devhub, reviewers-and-zadmin, users-accounts-and-ratings, main, ui-tests, flake8, docs, assets
```

Comment:
typo `amo-locales-and-signin` without the `g`. Does that `envlist` property matter, since it was completely missing a couple envs before ?

Reply:
nice find, the `envlist` is only used if you're running `tox` without any arguments / without specifying an environment. So it's only used by someone who uses tox locally.

Reply:
Should we drop the line then? If someone is using tox locally it's to run tests closer to how travis does, and if travis doesn't actually use this setting...